README.md

```diff
@@ -1,62 +1,28 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
+# First Mobile Version
 
 > Description the project.
+I choosed the frist template in Figma and modify a lot styles. This steps is very important for me to lear daily a lot things in the wordl of programming. 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
+-HTML/CCS.LINTERS
 
-### Setup
 
-### Install
 
-### Usage
+👤 **Lucca MIcale**
 
-### Run tests
+- GitHub: https://github.com/luccamicale
+- Twitter: https://twitter.com/LuccaMicale_
+- LinkedIn: https://www.linkedin.com/in/lucca-micale-907908233/
```

Comment:
Consider giving this section a title of  **`Authors`** as in the template provided by Microverse.